contracts/examples/Mars.sol

```diff
@@ -49,27 +49,6 @@ contract Mars is IbcReceiverBase, IbcReceiver {
         timeoutPackets.push(packet);
     }
 
-    function onCloseIbcChannel(bytes32 channelId, string calldata, bytes32) external virtual onlyIbcDispatcher {
-        // logic to determin if the channel should be closed
-        bool channelFound = false;
-        for (uint256 i = 0; i < connectedChannels.length; i++) {
```

Comment:
I think so, since we don't have support for this in our relayer. dropping this completely was @nicopernas 's suggestion, and I agree I don't see any downsides to dropping this since this isn't a finalized or supported method